docs/KVSet/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure App Configuration 'appconfig/kvset' profile
+Azure App Configuration can be exported and imported through azure CLI using the `appconfig/kvset` profile.
+
```

Comment:
Should there be a link to the schema file in the readme?

Reply:
I think it is a good idea! (y)